docs/INSTALL-devel.rst

```diff
@@ -0,0 +1,290 @@
+Following guide covers setting up a developement environment for HaaS
```

Comment:
We ( @kylehogan, @knikolla and you) decided not to worry about it too much right now.
We will have a separate PR for making things suitable for 'read the docs'